WordPress/src/main/res/values/strings.xml

```diff
@@ -2143,21 +2143,21 @@
 
     <!-- NUX strings -->
     <string name="invalid_email_message">Your email address isn\'t valid</string>
-    <string name="email_invalid" tools:ignore="UnusedResources">Enter a valid email address</string>
+    <string name="email_invalid" tools:ignore="UnusedResources" a8c-src-lib="wordpress-login-flow">Enter a valid email address</string>
     <string name="username_or_password_incorrect">The username or password you entered is incorrect</string>
```

Comment:
Interesting to see that `username_or_password_incorrect` for example does not come from the login library like I would initially thought… 🙃